src/features/utils/aggregate-base.js

```diff
@@ -47,6 +47,10 @@ export class AggregateBase extends FeatureBase {
     this.drained = true
   }
 
+  deregisterDrain () {
+    deregisterDrain(this.agentIdentifier, this.featureName)
+  }
+
```

Comment:
I think we should consider putting this on the `FeatureBase` class instead of the `AggregateBase` class so that the instrument modules can use this as well (Both AggregateBase and InstrumentBase extend FeatureBase so they would then both have access).  

There are a couple instrument files that call `deregisterDrain`:
* session_trace/instrument
* generic_events/instrument

Thoughts about that?

Reply:
That's fine with me.